helm/polaris/values.yaml

```diff
@@ -23,7 +23,7 @@ replicaCount: 1
 
 image:
   # -- The image repository to pull from.
-  repository: localhost:5001/polaris
+  repository: apache/polaris
```

Comment:
This is different, right?

Reply:
Different, but I think `apache/polaris` is more correct :)

Reply:
Exactly, `localhost:5001` is definitely not a good choice. This is only useful if you have a local registry running, but images with that kind of registry are otherwise unusable.

Since @jbonofre created our DockerHub space: https://hub.docker.com/r/apache/polaris, the official polaris image name will be `apache/polaris` - which expands to `docker.io/apache/polaris`.

This is the name I'm using consistently in this PR. E.g. if you build the images as of today with `./gradlew assemble -Dquarkus.container-image.build=true` you would get the following images loaded in your Docker environment:

* `apache/polaris:latest`
* `apache/polaris:1.0.0-incubating-SNAPSHOT`
* `apache/polaris-admin-tool:latest`
* `apache/polaris-admin-tool:1.0.0-incubating-SNAPSHOT`